cypress/TestCases/FireboltCertification/LifecycleTestcases/LifecycleTerminated.feature

```diff
@@ -5,7 +5,7 @@ Feature: Lifecycle_Terminated
         Given the environment has been set up for 'lifecycle' tests
         When 3rd party 'certification' app is launched with '<state>' state
         Then '3rd party app' invokes the 'Firebolt' API to 'call lifecycle terminate'
-        Then '3rd party app' is in 'terminated' state
+        Then '3rd party app' transitions to state 'terminated'
 
```

Comment:
The above line needs to be changed to the lifecycle transition glue (line 7) and not line 8 which is validation glue

Reply:
added the required changes in feature file